### ISSUE TITLE
audio examples

### ISSUE THREAD

**Issue by easypx** [PlayVoice/lora-svc#92]
https://vocaroo.com/16HthhIMS5ay 
Source: Dave Gahan Acapella
Singer: singer0015 
Model: maxgan_pretrain_32K.pth

Can you say, which gender the bundled singers have? I assume they are mixed male and female.

**Comment by MaxMax2016** [comment]
yes, they are mixed men and women. maxgan_pretrain_32K.pth is trained by the data: https://github.com/Multi-Singer/Multi-Singer.github.io

**Comment by easypx** [comment]
Thx, I'll take a look. (I've already figured out a lot by transposing.)
Here is the full mix of "Enjoy The Silence"! https://www.youtube.com/watch?v=3Y4UjhM0A6A

**Comment by easypx** [comment]
https://www.youtube.com/shorts/ou0atcGNXrs
Synthesized voice to custom model voice cloning. Model training took 3 nights (~30h in CPU mode). Proper CUDA hardware would be of advantage.
Software: Synthesizer V Studio Pro, PlayVoice/lora-svc

**Comment by easypx** [comment]
https://www.youtube.com/watch?v=Z0fdCtMGHsM
